policyengine_us/parameters/gov/states/vt/tax/income/agi/exclusions/capital_gain/flat/max_amount.yaml

```diff
@@ -0,0 +1,15 @@
+description: Vermont limits the flat capital gain exclusion at this amount.
```

Comment:
```suggestion
description: Vermont limits the flat capital gains exclusion at this amount.
```

Reply:
everywhere, I missed that but this is the phrasing from the form

Reply:
Should I change the variable names as well? or just the descriptions and labels.